core/src/components/button/button.scss

```diff
@@ -235,11 +235,6 @@
   @include margin(0, -0.2em, 0, 0.3em);
 }
 
-::slotted(ion-icon[slot="icon-only"]) {
-  font-size: 1.8em;
-}
```

Comment:
Removed since the font size is being overwritten through their respective mode files.